code/game/machinery/cloning.dm

```diff
@@ -144,6 +147,8 @@
 		var/mob/dead/observer/G = clonemind.get_ghost()
 		if(!G)
 			return FALSE
+		if(G.suiciding) // The ghost came from a body that is suiciding.
+			return FALSE

```

Comment:
Does this give any feedback? If not just transfer the suiciding var since the pod already checks for it (finally putting that useless check to use!)

Reply:
It gives you an `Initialisation failure.` on the console if you try to clone someone manually. Autoprocess just does nothing.

I'm not a fan of random unfinished clones popping out of the pod when someone suicides. I think it's better to leave it like this.

Reply:
@george99g I haven't read the code enough to determine how to make it more descriptive, although I guess it's not a big deal for the purpose of this PR.

Reply:
The easiest way I see is to make `growclone` return a number (with defines, obviously) to indicate the reason, which gets picked up by the calling code to show the right reason in the UI.
I don't want to deal with BYOND any more, though, so I'm just going to leave the PR as is.

Reply:
I don't blame you lol